queries/nu/highlights.scm

```diff
@@ -275,16 +278,12 @@ key: (identifier) @property
 (param_cmd
   (cmd_identifier) @string)
 
-(param_long_flag
-  (long_flag_identifier) @attribute)
-
-(param_short_flag
-  (param_short_flag_identifier) @attribute)
-
-(short_flag
-  (short_flag_identifier) @attribute)
-
-(long_flag_identifier) @attribute
+[
+  (attribute_identifier)
+  (long_flag_identifier)
+  (param_short_flag_identifier)
+  (short_flag_identifier)
+] @attribute
 
```

Comment:
@clason Does this look good?

Reply:
Almost (assuming you've tested against regressions, which I cannot since I don't know the language).

But the `"@"` should still be moved to after line 280.

Reply:
done, feels odd to me though, I mean why anonymous nodes are not put together.

Reply:
cool, I'm OK with any way.